styles/components/_layout.scss

```diff
@@ -425,3 +425,46 @@
 .posF {
   position: fixed;
 }
+.about { 
+  .brandSmall {
+    width: 50px;
+    hegiht: 50px;
+  }
+  .flexBar {
+    display: flex;
+    flex-direction: row;
+  }
+
+  .flexSpaceBetween {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+  }
+  
+  #donation-qr-code {
+    width: 200px; 
+    height: 200px;
+    background: black;
+  } 
+
+  #donate-cta {
+    max-width: 80%;
+    margin: auto;
+  }
+
+  .flex0 {
+    flex: 0;
+  }
+
+  .txtRight {
+    text-align: right;
+  }
+
+  .txtCent {
```

Comment:
We already have a txCtr class in _type.scss. Other generic type specific classes, like txtRight should also go there.